install_config/configuring_vsphere.adoc

```diff
@@ -235,3 +235,18 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
 [[vsphere-applying-configuration-changes]]
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[vsphere-backup]]
+== Backup of Persistent Volumes
+{product-title} provisions new volumes as *independent persistent disks* to
+freely attach and detach the volume on any node in the cluster. As a consequence,
+link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[it is not possible to back up volumes that use snapshots].
+
+[discrete]
```

Comment:
Can you remove this heading? We don't need the `Procedure` heading, but you could add an introductory sentence, such as "To back up PVs that use snapshots:"

Reply:
Heading removed, restored "To create a backup of PVs:".